Sources/EduIDExpansion/EduIDExpansion.swift

```diff
@@ -5,16 +5,19 @@ public final class EduIDExpansion: NSObject {
     public static let shared = EduIDExpansion()
     
     private var mainCoordinator: MainCoordinator!
-    
     private override init() {
         super.init()
     }
     
-    public func run() -> UINavigationController {
-        let navigationController = UINavigationController()
-        mainCoordinator = MainCoordinator(navigationController: navigationController)
+    public func attachViewController() -> UIViewController {
+        let homeViewController = HomeViewController()
+        let homeNavigationController = UINavigationController(rootViewController: homeViewController)
+        mainCoordinator = MainCoordinator(homeNavigationController: homeNavigationController)
+        homeViewController.delegate = mainCoordinator
```

Comment:
This logic belongs in the start method of the MainCoordinator.

Reply:
I did this